https://gitlab.gnome.org/GNOME/gtk/commit/5df4d277527858481ca6eb7af0762c73d804f19d

Fix build

broadway-server.c requires broadwayjs.h
this failed osc build -j1

Fixes #1218

Index: gtk+-3.94.0/gdk/broadway/meson.build
===================================================================
--- gtk+-3.94.0.orig/gdk/broadway/meson.build
+++ gtk+-3.94.0/gdk/broadway/meson.build
@@ -43,8 +43,19 @@ clienthtml_h = custom_target('clienthtml
   ],
 )
 
+broadwayjs_h = custom_target('broadwayjs.h',
+  input : ['broadway.js'],
+  output : 'broadwayjs.h',
+  command : [
+    gen_c_array,
+    '--array-name=broadway_js',
+    '--output=@OUTPUT@',
+    '@INPUT0@',
+  ],
+)
+
 libgdk_broadway = static_library('gdk-broadway',
-  clienthtml_h,
+  clienthtml_h, broadwayjs_h,
   gdk_broadway_sources, gdkconfig, gdkenum_h,
   include_directories: [confinc, gdkinc],
   c_args: [
@@ -58,17 +69,6 @@ libgdk_broadway = static_library('gdk-br
 
 broadwayd_syslib = os_win32 ? find_library('ws2_32') : shmlib
 
-broadwayjs_h = custom_target('broadwayjs.h',
-  input : ['broadway.js'],
-  output : 'broadwayjs.h',
-  command : [
-    gen_c_array,
-    '--array-name=broadway_js',
-    '--output=@OUTPUT@',
-    '@INPUT0@',
-  ],
-)
-
 executable('gtk4-broadwayd',
   clienthtml_h, broadwayjs_h,
   'broadwayd.c', 'broadway-server.c', 'broadway-output.c',
